### ISSUE TITLE
Averages Chart: clean up time selection (only one time is selectable for each time period

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#167]
```
Averages Chart: clean up time selection (only one time is selectable for each 
time period
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 30 Jun 2015 at 3:22

**Comment by lixun910** [comment]
In some cases (e.g. NAT with yearly time period), it it reasonable that only one time (one year) is selectable. 

However, in some cases (e.g. time period is weekly or monthly), users may need to select multiple time periods (e.g. first 4 weeks vs. next 4 weeks).

The current design (which allows very flexible selection on time) can handle both scenarios, so I suggest to keep this design for now.

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.85 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
